Source/CesiumRuntime/Private/CesiumEllipsoidFunctions.h

```diff
@@ -0,0 +1,63 @@
+// Copyright 2020-2024 CesiumGS, Inc. and Contributors
+
+#pragma once
+
+#include "Math/Matrix.h"
+
+namespace CesiumGeospatial {
+class Ellipsoid;
+};
+
+/**
+ * A collection of methods for working with {@link CesiumGeospatial::Ellipsoid}
+ * objects from Unreal.
+ */
+class CesiumEllipsoidFunctions {
```

Comment:
This a good way to separate shared functionality, but for code style, I'm don't see any other examples of `XxxFunctions` in the cesium-unreal code base.

There's always a first though.

Another optional way to do it is to just make these static functions in `CesiumEllipsoid`. `UCesiumWgs84Ellipsoid` should still be able to access that.

Reply:
It's used by `CesiumWgs84Ellipsoid` - since the existing code expects `UCesiumWgs84Ellipsoid` to have static methods for dealing with a WGS84 ellipsoid, our options are either 1. putting a bunch of static methods on `UCesiumEllipsoid` that accept an ellipsoid parameter, which both `UCesiumEllipsoid` and `UCesiumWgs84Ellipsoid` will use, 2. storing a static instance of a WGS84 `UCesiumEllipsoid` in `UCesiumWgs84Ellipsoid` which all the static methods use, or 3. creating static methods that accept an ellipsoid parameter and using them for both `UCesiumEllipsoid` and `UCesiumWgs84Ellipsoid` (which is what I went with here).